enginetest/enginetests.go

```diff
@@ -4996,6 +4996,80 @@ func TestAddDropPks(t *testing.T, harness Harness) {
 			{1, 1},
 		}, nil, nil)
 	})
+
+	if _, ok := harness.(*MemoryHarness); ok {
```

Comment:
Don't skip like this. Engine tests should always be harness agnostic.

Make a new test method called `TestAddAutoIncrementColumn` and skip it from `memory_engine_test.go`